src/main/java/com/hotel/interceptors/ApiKeyAuthInterceptor.java

```diff
@@ -1,19 +1,18 @@
 package com.hotel.interceptors;
 
-import io.grpc.ServerInterceptor;
-import io.grpc.ServerCall;
-import io.grpc.Status;
-import io.grpc.Metadata;
-import io.grpc.ServerCallHandler;
+import com.hotel.util.APIConstants;
+import io.grpc.*;
 import io.jsonwebtoken.SignatureAlgorithm;
```

Comment:
remove this *import, use what was existing

Reply:
this was done as part of sonar lint, already resolved